scripts/edit-task.js

```diff
@@ -0,0 +1,231 @@
+import { CLICK_EVENT, HIDE_CLASS } from "/scripts/constants.js";
+import createButton from "/scripts/factory/createButton.js";
+import toggleButtons from "/scripts/factory/toggleButton.js";
+
+function saveEvent(
+  doneButton,
+  editButton,
+  deleteButton,
+  saveAndDoneButton,
+  revertButton,
+  saveButton,
+  inputElement,
+  tasks,
+  id,
+  saveEventHandler,
+  saveAndDoneEventHandler,
+  revertEventHandler
+) {
+  inputElement.contentEditable = false;
+  tasks[id].content = inputElement.textContent;
+  toggleButtons(
+    editButton,
+    deleteButton,
+    revertButton,
+    saveButton,
+    doneButton,
+    saveAndDoneButton
+  );
+  console.log("call");
```

Comment:
log statements should not be committed. 

```suggestion
  );
```